conans/model/version_range.py

```diff
@@ -56,9 +56,14 @@ def first_non_zero(main):
         else:
             return [_Condition(operator, Version(version))]
 
-    def valid(self, version):
+    def valid(self, version, conf_resolve_prepreleases=None):
```

Comment:
``def valid`` happens to be used only internally in ``VersionRange``, it can be made private (I was concerned about the ``=None`` default too)